internal/tools.go

```diff
@@ -18,7 +18,3 @@
 // This is used to import things required by build scripts, to force `go mod` to see them as dependencies
 
 package internal
```

Comment:
Do we need this at all now?

Reply:
I'm ok with it existing until the official go blessed way of doing it lands, in case we want to put stuff here.